src/scripts/docs/examples/DropdownMultiSelectAdvanced.example.txt

```diff
@@ -0,0 +1,82 @@
+class ComponentExample extends React.Component {
+  state = {
+    selectedItems: {},
+    selectedItemsIds: [],
+    itemSearchLoading: false,
+    items: {
+          1: { id: 1, title: 'Ben Bruning', name: 'BB', profileImageUrl: '' },
+          2: { id: 2, title: 'Blake Guilloud', name: 'BG', profileImageUrl: '' },
+          3: { id: 3, title: 'Craig Anthony', name: 'CA', profileImageUrl: '' },
+          4: { id: 4, title: 'Grady Rogers', name: 'GR', profileImageUrl: '' },
+          5: { id: 5, title: 'Keaton Foster', name: 'KF', profileImageUrl: '' },
+          6: { id: 6, title: 'Lauren Farr', name: 'LF', profileImageUrl: '' },
+          7: { id: 7, title: 'Keaton Foster',name: 'KF', profileImageUrl: '' },
+          8: { id: 8, title: 'Paul Griffin', name: 'PG', profileImageUrl: '' },
+          9: { id: 9, title: 'Rob Whelan', name: 'RW', profileImageUrl: '' },
+          10: { id: 10, title: 'Terry Kennair', name: 'TK', profileImageUrl: '' },
+        },    
+    itemsIds: [],
+    itemsCopy: {},    
+  }
+
+  componentWillMount(){
+    this.setState({ itemsIds: Object.keys(this.state.items), itemsCopy: this.state.items });
+  }
+
+  fetchAllItems = (searchText) => {
```

Comment:
Try not to use `fetch` when you're not making a network request or getting data from an external resource.

Reply:
@alexgsummer In case of our members and contacts filters dropdown in rhinofront, I will be fetching data from api, that's why I used it.